applications/DeepSpeed-Chat/training/utils/data/raw_datasets.py

```diff
@@ -341,6 +341,52 @@ def get_prompt_and_rejected(self, sample):
         )
         return None
 
+# Chinese dataset
```

Comment:
Don't need this comment since this is not language specific

Reply:
Thanks for figureing this out. I really forgot this.